model/Core/Properties/algorithm.md

```diff
@@ -4,11 +4,11 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+Specifies the algorithm used for calculating the hash value.
```

Comment:
If we change the property `hash` to `checksum` we may want to use `checksum` rather than hash - same comment applies to the use of hash in other descriptions and files